nbconvert/exporters/latex.py

```diff
@@ -43,6 +43,10 @@ def _template_skeleton_path_default(self):
     template_extension = Unicode(".tplx").tag(config=True)
 
     output_mimetype = 'text/latex'
+    
+    def default_filters(self):
+        yield from super(TemplateExporter,self).default_filters()
```

Comment:
Not sure off the top of my head. If you're interested in working it out, we can try to debug on the Hacktrain tomorrow.